### DISCUSSION TITLE
compile on save w/ network drive

### DISCUSSION THREAD

**Discussion by mrjameswells** [glenn2223/vscode-live-sass-compiler#245] [Q&A] [answered]
How to get the extension's compile on save feature to work with mapped network drives?

**Comment by glenn2223** [comment]
I'm not sure what you mean. If the VSCode workspace is operating inside the network drive then it should recognise a save and trigger compilation

**Reply by glenn2223** [reply]
Can you expand and maybe provide screenshots?

**Comment by glenn2223** [comment, marked as answer]
Have you tried using an SSH connection, as per https://github.com/glenn2223/vscode-live-sass-compiler/discussions/254
